flexget/plugins/sites/lostfilm.py

```diff
@@ -0,0 +1,179 @@
+
+__author__ = 'danfocus'
+
+log = logging.getLogger('lostfilm')
+
+EPISODE_REGEXP = re.compile(
+    '.*lostfilm.tv/series/.*/season_(\d+)/episode_(\d+)/.*')
+LOSTFILM_ID_REGEXP = re.compile(
+    '.*static.lostfilm.tv/Images/(\d+)/Posters/poster.jpg.*')
+TEXT_REGEXP = re.compile('^(\d+)\sсезон\s(\d+)\sсерия\.\s(.+)\s\((.+)\)$')
+
+quality_map = {
+    'SD': ('480p', 'avi'),
+    '1080': ('1080p', 'mkv'),
+    'MP4': ('720p', 'mp4'),
+    'HD': ('720p', 'mkv')
+}
+
+LOSTFILM_URL = 'http://lostfilm.tv/rss.xml'
+
+
+class LostFilm(object):
+    """
+    Change new lostfilm's rss links
+
+    Example::
+
+      lostfilm: yes
+
+    Advanced usage:
+
+      lostfilm:
+        url: <url>
+    """
+
+    schema = {
+        'type': ['boolean', 'object'],
+        'properties': {
+            'url': {'type': 'string', 'format': 'url'},
+        },
+        'additionalProperties': False
+    }
+
+    def build_config(self, config):
+        """Set default url to config"""
+        if isinstance(config, bool) and config is True:
+            config = {'url': LOSTFILM_URL}
+        return config
+
+    def on_task_input(self, task, config):
+        config = self.build_config(config)
+        if config is False:
+            return
+        rss = feedparser.parse(config['url'])
+        if rss.get('status') != 200:
+            log.error('Received not 200 (OK) status')
+            return
+        entries = []
+        for item in rss.entries:
+            if item.get('link') is None:
+                log.debug('Item doesn\'t have a link')
+                continue
+            if item.get('description') is None:
+                log.debug('Item doesn\'t have a description')
+                continue
+            match = LOSTFILM_ID_REGEXP.search(item['description'])
+            if match:
+                lostfilm_num = match.group(1)
+            else:
+                log.debug('Item doesn\'t have lostfilm id in description')
+                continue
+            match = EPISODE_REGEXP.match(item['link'])
+            if match:
+                season_num = int(match.group(1))
+                episode_num = int(match.group(2))
+            else:
+                log.debug('Item doesn\'t have episode id in link')
+                continue
+            redirect_url = 'http://www.lostfilm.tv/v_search.php?' \
+                'c={}&s={}&e={}'.format(lostfilm_num, season_num, episode_num)
+            try:
+                response = task.requests.get(redirect_url)
+            except RequestException as e:
+                log.error('Could not connect to redirect url: {:s}'.format(e))
+                continue
+
+            page = get_soup(response.content)
+            try:
+                redirect_url = page.head.meta['content'].split('url=')[1]
+            except:
+                log.error('Missing redirect')
+                continue
+
+            try:
+                response = task.requests.get(redirect_url)
```

Comment:
It's allright.
v_search.php have in head with redirect url
<meta content="0; url=http://retre.org/v3/?c=288&amp;s=1&amp;e=14&amp;u=XXXXXX&amp;h=XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX&amp;n=1" http-equiv="refresh"/>